src/functions/asyncCredential/sessionService/sessionService.ts

```diff
@@ -11,7 +13,7 @@ import {
   successResponse,
 } from "../../types/errorOrValue";
 
-export class SessionService implements IRecoverAuthSession {
+export class SessionService implements ISessionService {
```

Comment:
If we're using an interface, may we please target it to the function?

E.g. ICreateSession

Reply:
Done